fix(docs): edited typos on reemo tutorial MTA-1828 (#459)
scaleway/docs-content: tutorials/access-mac-mini-with-reemo/index.mdx

## Reeemo - Overview

[Reemo](https://reemo.io/) is an ultra-low latency, high performance desktop platform designed to allow anyone to connect to any computer and virtual environment from a Chrome browser to work and play freely from anywhere, on any device. 

> Reemo works perfectly on MacOS, Linux, and Windows and supports multiple peripherals. _Reemo is a French startup that was part of the Scaleway Startup Program._

Reviewer: ```suggestion
Reemo works perfectly on MacOS, Linux, and Windows and supports multiple peripherals. _Reemo is a French startup that participates in the [Scaleway Startup Program](https://www.scaleway.com/en/startup-program/)._
```

Author: According to the Startup Program confluence page, Reemo no longer participates in the Startup Program.
